### PR TITLE
Added Numble and Quick tests

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
+//  EmptyViewSpy.swift
+//  BootstrapTests
+//
+//  Created by Alex Yanski on 3/25/20.
+//  Copyright © 2020 Alex Yanski. All rights reserved.
+//
+
+import UIKit
```
Comment (pkondrashkov) [inline BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Is this needed?

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+@testable import Bootstrap
+
+final class EmptyViewSpy: EmptyView {
```
Comment (pkondrashkov) [inline BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Please clean up spies so they have only fields that are used by the test case. We are not gonna use mock generator so there is no need to use it here, let's pretend like you wrote this spy class on your own.

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift (hunk 1 above)
Comment (AlexYanski) [reply BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Yes, there is a UIImage

Comment (pkondrashkov) [reply BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Hmm I see. Thanks

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift (hunk 2 above)
Comment (AlexYanski) [reply BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Done

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    var invokedDisplayHeaderTitleParameters: (headerTitle: String, Void)?
```
Comment (pkondrashkov) [inline BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: (headerTitle: String, Void)? can be replaced with just  : String?

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    func display(headerTitle: String) {
+        invokedDisplayHeaderTitleParameters = (headerTitle, ())
+    }
+
+    var invokedDisplayFooterImageParameters: (footerImage: UIImage, Void)?
```
Comment (pkondrashkov) [inline BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: same

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift (hunk 3 above)
Comment (AlexYanski) [reply BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Done

File: BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift (hunk 4 above)
Comment (AlexYanski) [reply BootstrapTests/Empry/TestDoubles/EmptyViewSpy.swift]: Done

**Pull request merged**